resin/models/application.py

```diff
@@ -56,10 +57,13 @@ def get(self, name):
             'eq': name
         }
         try:
-            return self.base_request.request(
+            apps = self.base_request.request(
                 'application', 'GET', params=params,
                 endpoint=self.settings.get('pine_endpoint')
-            )['d'][0]
+            )['d']
+            if len(apps) > 1:
```

Comment:
Yeah, I will add support for both name and id for application and device soon since we now need to prioritize Python 3 SDK support.